frontend/src/views/Chat.vue

```diff
@@ -1,13 +1,263 @@
+                        optionValue="code" placeholder="Choose dictionary..." class="w-fit max-w-12rem md:max-w-max h-fit m-2" />
+                <Button severity="secondary" icon="pi pi-info" rounded class="m-2" />
+    
+                <Dropdown v-model="selectedDbms" :options="dbms" optionLabel="name" optionValue="code"
+                    class="w-fit h-fit m-2" />
+                <Dropdown v-model="selectedLanguage" :options="languages" optionLabel="name" optionValue="code"
+                    class="w-fit h-fit m-2" />
+            </div>
+        </Panel>-->
+
+        <!-- CHAT MESSAGES -->
+        <div id="messages">
+            <div class="message sent md:w-10 mx-1 my-2">
+                <div class="flex flex-row-reverse gap-3">
+                    <div class="flex-shrink-0">
+                        <Avatar icon="pi pi-user" size="large" shape="circle" />
+                    </div>
+                    <div class="w-full border-round-lg messageBox">
+                        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Autem unde tenetur labore accusantium quae
+                        quia inventore quisquam eligendi aliquid doloremque qui amet sit et cum cupiditate consectetur quo
+                        rerum maiores adipisci assumenda impedit, sequi excepturi. Adipisci, in. Similique, natus illo
+                        facere, quas, ea optio saepe architecto unde aliquid vero itaque!</p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="message received md:w-10 mx-1 my-2">
+                <div class="flex gap-3">
+                    <div class="flex-shrink-0">
+                        <Avatar icon="pi pi-database" class="" size="large" shape="circle" />
+                    </div>
+                    <div class="w-full border-round-lg messageBox">
+                        <!--<Button :label="isCopying ? '' : 'Copy'" :icon="isCopying ? 'pi pi-check' : 'pi pi-copy'" class="copy-to-clipboard align-self-end mt-2 mr-2" severity="contrast" @click="copyToClipboard"/>-->
+                        <Button :icon="isCopying ? 'pi pi-check' : 'pi pi-copy'" class="copy-to-clipboard" severity="contrast" @click="copyToClipboard" aria-label="Copy to clipboard" />
+                        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusantium voluptatem soluta quidem ea sit, 
+                        quisquam unde dolor dicta temporibus error.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <!-- INPUT select e prompt -->
+        <div id="input-container" class="bottom-0 left-0 right-0 flex items-end">
+            <Textarea v-model="request" autoResize placeholder="Enter a natural language request" rows="1"
+                class="w-full" />
+            <Button @click="runRequest" aria-label="Send" icon="pi pi-send" rounded />
```

Comment:
aggiungerei un componenete `InputGroup` per attaccare textarea e pulsante di invio
```suggestion
            <InputGroup >
                <Textarea v-model="request" autoResize placeholder="Enter a natural language request" rows="1"
                class="w-full" :disable="!selectedDictionary" />
                <Button @click="runRequest" label="Send" icon="pi pi-send" :disabled="!selectedDictionary" />
            </InputGroup>
```

Reply:
Questo ci sta, direi di aggiungerlo sicuramente